Only include the local macOS 11 SDK copy of vecLib.h when using the
macOS 12 SDK or later to prevent build failure when using the OS X 10.9
SDK or older.
https://github.com/mcg1969/vecLibFort/issues/14
https://github.com/mcg1969/vecLibFort/pull/15
--- vecLibFort.c.orig	2022-01-22 08:32:18.000000000 -0600
+++ vecLibFort.c	2022-02-08 19:04:29.000000000 -0600
@@ -19,9 +19,11 @@
 /* Don't load the CLAPACK header, because we are using a different calling
    convention for the replaced functions than the ones listed there. */
 #define __CLAPACK_H
+#include <AvailabilityMacros.h>
+#if MAC_OS_X_VERSION_MAX_ALLOWED >= 120000
 #include "vecLib-760.100.h"
+#endif
 #include <Accelerate/Accelerate.h>
-#include <AvailabilityMacros.h>
 
 /* Add a SGEMV fix for Mavericks. See
   http://www.openradar.me/radar?id=5864367807528960 */
